### PR TITLE
linux_testing_bcachefs: 5.8.0-2020.09.07 -> 5.9.0-2020.11.17 // bcachefs-tools: 2020-08-25 -> 2020-11-17

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change
Update bcachefs

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Chiiruno** [comment]
cc @davidak

**Review by davidak — APPROVED** [review]
Looks good

not tested

**Comment by davidak** [comment]
I successfully build the kernel, booted into it, formatted a filesystem and mounted it.

File: pkgs/tools/filesystems/bcachefs-tools/default.nix [hunk 1]
```diff
@@ -22,34 +22,22 @@ stdenv.mkDerivation {
                 "INITRAMFS_DIR=${placeholder "out"}/etc/initramfs-tools"
   '';
 
-  enableParallelBuilding = true;
-
-  nativeBuildInputs = [
-    pkgconfig
-  ];
+  nativeBuildInputs = [ pkgconfig ];
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/bcachefs-tools/default.nix:25]: this should be pkg-config.

Comment (AndersonTorres) [reply pkgs/tools/filesystems/bcachefs-tools/default.nix:25]: Treewide substitution time!

Comment (SuperSandro2000) [reply pkgs/tools/filesystems/bcachefs-tools/default.nix:25]: Yes, please!

**Pull request merged**